metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/writer/SimpleHbaseEnrichmentWriter.java

```diff
@@ -169,16 +188,19 @@ public HTableInterface getTable(Map<String, Object> config) throws IOException {
   private List<String> getColumns(Object keyColumnsObj, boolean allowNull) {
     Object o = keyColumnsObj;
     if(allowNull && keyColumnsObj == null) {
+      LOG.debug("No key columns were specified");
       return Collections.emptyList();
     }
     if(o instanceof String) {
+      LOG.debug("Key column: '" + o + "'");
```

Comment:
Thanks everyone for a great feedback! @justinleet I will rebase once #599 is merged, cheers.